.github/workflows/run_periodic_tests.yml

```diff
@@ -129,3 +129,48 @@ jobs:
         if: matrix.os == 'ubuntu-latest'
         run: |
           tox -e odes
+
+  #M-series Mac Mini
+  build-apple-mseries:
+      needs: style
+      runs-on: [self-hosted, macOS, ARM64]
+      strategy:
+        fail-fast: false
+        matrix:
+          python-version: ["3.8", "3.9", "3.10", "3.11"]
+          
+      steps:
+        - uses: actions/checkout@v3
+        - name: Install python & create virtualenv
+          shell: bash
+          run: |
+            export PATH="$PYENV_ROOT/bin:$PATH"
```

Comment:
should put PATH in `env` (https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsenv)